### ISSUE TITLE
terra-table with chevrons shows chevron on disabled row

### ISSUE THREAD

**Issue by sparkym3** [cerner/terra-core#3138]
# Bug Report

## Description
When a disclose table with chevrons has a disabled row, that row is still showing the chevron which would indicate it is selectable.

## Expected Behavior
Never show chevron on disabled rows.

## Possible Solution
Check the disabled flag when flexing the chevron display.

## Environment
terra-table version 4.15.0

**Comment by jeremyfuksa** [comment]
Looking at the doc site examples and playing with disabling the chevron row, I've got a handle on what you're experiencing and have some feedback.

While the table row subcomponent has an `isDisabled` prop, it does not have corresponding visual styling to support that. That's because we can't know what the full content of your row is going to be. So, it's on the consumer to style disabled table rows of any type properly, based on their unique needs. Ideally, the consuming team has access to a UX designer that can help determine what that styling needs to be.

For the chevron example, a lightening of the chevron is preferable to the removal of it. Removal of the chevron infers that interactability is no longer possible, while that may not be completely accurate (e.g. temporary disabling while waiting for services to return data, etc.).

As for clear-cut guidance and standards for how disabled disclosure rows should look, that's still a work in progress on the UX Foundations side of this equation.
